src/server/svr_jobfunc.c

```diff
@@ -4131,6 +4131,10 @@ change_enableORstart(resc_resv *presv, int which, char *value)
 	if (presv->ri_qs.ri_type != RESC_RESV_OBJECT)
 		return (0);
 
+	if (which == Q_CHNG_START && strcmp(value, ATR_TRUE) == 0 &&
+		! presv->ri_wattr[RESV_ATR_resv_nodes].at_flags & ATR_VFLAG_SET)
+		return (0);
+
```

Comment:
It means: Never start a reservation queue if its reservation is without RESV_ATR_resv_nodes.

If such a reservation queue is started (the associated RESV_ATR_resv_nodes is not set) then the associated jobs could run on any node!

It covers the case when the maintenance wipes the whole RESV_ATR_resv_nodes of common reservation and the reservation is degraded but it will start anyway.

Reply:
Ahh got it.  Thanks for the explanation.